sass/directives/_boxes.scss

```diff
@@ -10,3 +9,23 @@
   padding: $base-padding;
   background-color: $panel-primary;
 }
+
+// Accordion Panels
+@mixin accordion__panel {
+    padding: 0 18px;
+    background-color: $white;
+    max-height: 0;
```

Comment:
I think you're using `max-height: 0` and `500` to show and hide the accordion. Did you try `display: none` and `display: block`? that way the height can automatically adjust to the amount of text